### PR TITLE
Simplifies creating import- an export jobs

### PR DESCRIPTION
**Pull request by andyHa**

### CODE REVIEW

File: src/main/java/sirius/biz/jobs/batch/file/EntityImportJobFactory.java [hunk 1]
```diff
@@ -40,7 +41,31 @@ protected EntityImportJobFactory() {
     }
 
     @Override
-    protected abstract EntityImportJob<?> createJob(ProcessContext process);
+    @SuppressWarnings("squid:S2095")
+    @Explain("The job must to ne closed here as it is returned and managed by the caller.")
```
Comment (sabieber) [inline src/main/java/sirius/biz/jobs/batch/file/EntityImportJobFactory.java]:
```suggestion
    @Explain("The job must not be closed here as it is returned and managed by the caller.")
```

File: src/main/java/sirius/biz/codelists/mongo/MongoCodeListImportJobFactory.java [hunk 2]
```diff
@@ -30,40 +30,33 @@
  */
 @Register(classes = JobFactory.class, framework = MongoTenants.FRAMEWORK_TENANTS_MONGO)
```
Comment (sabieber) [inline src/main/java/sirius/biz/codelists/mongo/MongoCodeListImportJobFactory.java]: Nope

File: src/main/java/sirius/biz/codelists/jdbc/SQLCodeListExportJobFactory.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+@Permission(CodeListController.PERMISSION_MANAGE_CODELISTS)
+public class SQLCodeListExportJobFactory
+        extends EntityExportJobFactory<SQLCodeListEntry, SmartQuery<SQLCodeListEntry>> {
+
+    private CodeListParameter codeListParameter = new CodeListParameter("codeList", "$CodeList").markRequired();
+
+    @Nonnull
+    @Override
+    public String getName() {
+        return "export-sql-code-list-entries";
```
Comment (sabieber) [inline src/main/java/sirius/biz/codelists/jdbc/SQLCodeListExportJobFactory.java]: Are configs needed for the new jobs?

Comment (andyHa) [reply src/main/java/sirius/biz/codelists/jdbc/SQLCodeListExportJobFactory.java]: configs?

**Pull request merged**